packages/dart/noports_core/lib/src/npa/npa_impl.dart

```diff
@@ -91,42 +101,95 @@ class NPAImpl with AtClientBindings implements NPA {
 
   @override
   Future<void> run() async {
-    AtRpc rpc = AtRpc(
+    _startPolicyInfoRpcServer();
+
+    subscribe(regex: r'.*\.devices\.policy\.sshnp', shouldDecrypt: true).listen(
```

Comment:
Newer daemons will ping their policy service to get some additional config information. For now that's just `eventLoggingConfig` but this has been written to be extensible as and when we have additional config info to send